packages/bpk-component-calendar/src/Week.js

```diff
@@ -87,6 +102,30 @@ class Week extends Component {
     return false;
   }
 
+  componentWillUnmount() {
+    document.removeEventListener('mouseup', this.onMouseUp);
+  }
+
+  onMouseClick(date) {
+    this.props.onDateClick(date);
+    this.setState({ savedDate: null });
+  }
+
+  onMouseDown(date, month) {
+    if (isSameMonth(date, month)) {
+      this.props.onDateMouseDown(null, { date });
+    } else {
+      this.setState({ savedDate: date });
```

Comment:
I'm trying to understand why this should be different. Can you expand on the reason?